src/Microsoft.Extensions.Configuration.AzureAppConfiguration/EnvironmentVariables.cs

```diff
@@ -14,12 +14,18 @@ internal static class EnvironmentVariables
         /// When set to "true", schema compatibility checks for feature flags are disabled,
         /// and all feature flags will be interpreted using the Microsoft Feature Flags schema.
         /// </summary>
-        public const string DisableFmSchemaCompatibility = "AZURE_APP_CONFIGURATION_FM_SCHEMA_COMPATIBILITY_DISABLED";
+        public const string DisableFmSchemaCompatibilityKey = "AZURE_APP_CONFIGURATION_FM_SCHEMA_COMPATIBILITY_DISABLED";
```

Comment:
Change the class name to EnvironmentVariableNames and then `Key` isn't needed at the end of all the fields.

Reply:
But if we avoid any changes ([comment](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/700#discussion_r2388728958)), then I see no need for the change